packages/fluentui/react-component-ref/src/RefForward.tsx

```diff
@@ -20,7 +20,7 @@ export class RefForward extends React.Component<RefProps> {
   }
 
   componentWillUnmount() {
-    delete this.currentNode;
+    this.currentNode = null;
```

Comment:
@miroslavstastny this part was a fix of memory leak. Is this change safe?

Reply:
Unless there's some nuance that I'm not aware of, I thought that removing the reference by either deleting or setting to another value should work to address memory leaks.

Reply:
Agree with @ecraig12345 - memory-leak wise, the two should be equal.